src/quickgui/qgsquickcoordinatetransformer.cpp

```diff
@@ -0,0 +1,100 @@
+/***************************************************************************
+ qgsquickcoordinatetransformer.cpp
+  --------------------------------------
+  Date                 : 1.6.2017
+  Copyright            : (C) 2017 by Matthias Kuhn
+  Email                :  matthias (at) opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsquickcoordinatetransformer.h"
+
+QgsQuickCoordinateTransformer::QgsQuickCoordinateTransformer( QObject *parent )
+  : QObject( parent )
+{
+  mCoordinateTransform.setSourceCrs( QgsCoordinateReferenceSystem::fromEpsgId( 4326 ) );
+  mCoordinateTransform.setContext( QgsCoordinateTransformContext() );
```

Comment:
Arghhh -- never do this! I'd make setDestinationCrs require a compulsory QgsCoordinateTransformContext argument instead and use that

Reply:
@nyalldawson  I was looking for that one too.... I wonder if we should not get a warning when someone adds a call like this. Either at compile time or with a code layout test. Thoughts?